client/src/components/leftSidebar/topLeftLogoAndTitle.js

```diff
@@ -3,6 +3,7 @@ import React from "react";
 import { connect } from "react-redux";
 import * as globals from "../../globals";
 import Logo from "../framework/logo";
+import Truncate from "../util/truncate";
```

Comment:
Uber nit: Should we change the file name to `Truncate.js` to signal that it's a React Component instead of a plain old JS function?

Reply:
Good q! We have not adopted this pattern elsewhere in the codebase, so it would mean renaming lots of files. Prefer to discuss separately and leave this lowercase for now (ie., if we decide to change, change all at once).

Reply:
SGTM!